nautobot/dcim/tables/devices.py

```diff
@@ -544,6 +547,7 @@ class Meta(ModularDeviceComponentTable.Meta):
             "type",
             "maximum_draw",
             "allocated_draw",
+            "power_factor",
```

Comment:
Not sure I agree with this being a default column, especially if it'll be left as 0.95 in the majority of cases. Thoughts?